kolibri/plugins/learn/assets/src/views/content-page/index.vue

```diff
@@ -106,7 +115,7 @@
         return this.$tr('recommended');
       },
       progress() {
-        if (this.userkind.includes(UserKinds.LEARNER)) {
+        if (!this.userkind.includes(UserKinds.SUPERUSER)) {
```

Comment:
this will likely have a conflict with the report-details branch for the reason stated above. recommend merging and retargeting to that branch